### PR TITLE
Add README

### PR DESCRIPTION
**Pull request by Phil-Friderici**

### CODE REVIEW

**Comment by Phil-Friderici** [comment]
to be continued...

**Comment by Phil-Friderici** [comment]
to be continued...

**Comment by Phil-Friderici** [comment]
Oh dear, now it fails on Ruby 1.8.7. Will look into this later.

File: Gemfile [hunk 1]
```diff
@@ -27,3 +26,6 @@ gem 'rspec',     '~> 2.0'   if RUBY_VERSION >= '1.8.7' && RUBY_VERSION < '1.9'
 gem 'rake',      '~> 10.0'  if RUBY_VERSION >= '1.8.7' && RUBY_VERSION < '1.9'
 gem 'json',      '<= 1.8'   if RUBY_VERSION < '2.0.0'
 gem 'json_pure', '<= 2.0.1' if RUBY_VERSION < '2.0.0'
+
+# metadata-json-lint fails on Ruby 1.8.7 since 0.0.19, only use it on Ruby 1.9 and higher
+gem 'metadata-json-lint' if RUBY_VERSION >= '1.9'
```
Comment (ghoneycutt) [inline Gemfile]:
```ruby
gem 'metadata-json-lint', '0.0.18' if RUBY_VERSION < '1.9'
gem 'metadata-json-lint', if RUBY_VERSION >= '1.9'
```

**Comment by ghoneycutt** [comment]
I'm seeing green! Is this good to merge?

**Comment by Phil-Friderici** [comment]
docs for the defines are still missing

File: Gemfile (hunk 1 above)
Comment (Phil-Friderici) [inline Gemfile]: Awesome! Works well without the comma in the last line:
```ruby
gem 'metadata-json-lint', '0.0.11'   if RUBY_VERSION < '1.9'
gem 'metadata-json-lint'             if RUBY_VERSION >= '1.9'
```

**Comment by Phil-Friderici** [comment]
good to merge now :)

**Comment by ghoneycutt** [comment]
Thank you!

**Pull request merged**